wiki/en/misc/general.yml

```diff
@@ -1,5 +1,5 @@
 footer:
-  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>
+  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a> - want to <a href="/wiki/Contribution">get involved?</a> <p> <img alt="ASIO logo" src="/assets/img/asio.jpg" id="asio" loading="lazy"> <br><em>ASIO is a trademark and software of Steinberg Media Technologies GmbH</em>
```

Comment:
I'm not sure if this looks ok in the footer. I'd rather create another FOX-CSS container which maybe also links to JACK...